assistant/src/main/java/com/ibm/watson/assistant/v1/model/WorkspaceSystemSettingsDisambiguation.java

```diff
@@ -38,6 +38,11 @@
   private String noneOfTheAbovePrompt;
   private Boolean enabled;
   private String sensitivity;
+  private Boolean randomize;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)